### PR TITLE
Feature/df 794 stelle beanstanden

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/complaint/ComplaintApplicationService.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    public ComplaintApplicationService(MailSenderService mailSenderService, ComplaintProperties complaintProperties, JobAdvertisementRepository jobAdvertisementRepository) {
+        this.mailSenderService = mailSenderService;
+        this.complaintProperties = complaintProperties;
+        this.jobAdvertisementRepository = jobAdvertisementRepository;
+    }
+
+    public void sendComplaint(ComplaintDto complaintDto) {
+        Condition.notNull(complaintDto);
+        Condition.notEmpty(complaintDto.getJobAdvertisementId());
+        JobAdvertisement jobAdvertisement = Condition.notNull(getJobAdvertisement(new JobAdvertisementId(complaintDto.getJobAdvertisementId())));
```
Comment (cjpsycos) [inline application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/complaint/ComplaintApplicationService.java]: Condition not needed due the fact getJobadvertisement throws an exception if not found

File: infrastructure/src/main/resources/mails/i18n/mail-messages.properties [hunk 2]
```diff
@@ -69,6 +69,20 @@ mail.jobAd.cancelled.body.addressblock.tel_it=Tel:
 mail.jobAd.cancelled.body.addressblock.fax_it=Fax:
 mail.jobAd.cancelled.body.addressblock.email_it=Email:
 
+mail.complaint.body=Im Job-Room wurde eine Stelle beanstandet
```
Comment (cjpsycos) [inline infrastructure/src/main/resources/mails/i18n/mail-messages.properties]: mail.complaint.body.introduction

File: infrastructure/src/main/resources/mails/i18n/mail-messages.properties [hunk 3]
```diff
@@ -69,6 +69,20 @@ mail.jobAd.cancelled.body.addressblock.tel_it=Tel:
...
+mail.complaint.body2=Stelle
```
Comment (cjpsycos) [inline infrastructure/src/main/resources/mails/i18n/mail-messages.properties]: mail.complaint.body.jobad

File: infrastructure/src/main/resources/mails/i18n/mail-messages.properties [hunk 4]
```diff
@@ -69,6 +69,20 @@ mail.jobAd.cancelled.body.addressblock.tel_it=Tel:
...
+mail.complaint.jobad.sourcesystem=Herkunft: {0}
+mail.complaint.jobad.id=ID: {0}
+mail.complaint.jobad.stellennummeregov=Stellennummer EGOV: {0}
+mail.complaint.jobad.stellennummeravam=Stellennummer AVAM: {0}
+mail.complaint.body3=Melder
```
Comment (cjpsycos) [inline infrastructure/src/main/resources/mails/i18n/mail-messages.properties]: mail.complaint.body.complainer

File: application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/complaint/ComplaintApplicationService.java (hunk 1 above)
Comment (fabiangotzen) [reply application/src/main/java/ch/admin/seco/jobs/services/jobadservice/application/complaint/ComplaintApplicationService.java]: Removed the condition & improved the naming of the mail keys.

**Pull request merged**